javascripts/discourse/components/custom-navigation.js

```diff
@@ -7,12 +7,31 @@ export default Component.extend({
   router: service(),
   tagName: "",
 
+  @discourseComputed("categoriesLoaded", "allowedPage")
+  shouldShow(categoriesLoaded, allowedPage) {
+    if (categoriesLoaded && allowedPage) {
+      document.body.classList.add("sidebar-navigation-shown");
+      return true;
+    } else {
+      document.body.classList.remove("sidebar-navigation-shown");
+      return false;
+    }
+  },
+
   @discourseComputed()
   categoriesLoaded() {
     if (this.siteSettings.login_required && !this.currentUser) return false;
     return Category.list().length !== 0;
   },
 
+  @discourseComputed("router.currentRoute")
+  allowedPage(currentRoute) {
+    return (
+      !currentRoute?.name.includes("user") &&
+      !currentRoute?.name.includes("admin")
```

Comment:
Maybe something along those lines would be better, to avoid any potential "false positives"?

```suggestion
      !currentRoute?.name.startsWith("user.") &&
      !currentRoute?.name.startsWith("admin.")
```

Reply:
I added these instead of the `starsWith` because I noticed that on some of the pages I did not want it rendering on, included user, but not in the specific `user.` way. Does that make sense?